apps/snitch_core/lib/core/tools/mailer/order_confirmation_email.ex

```diff
@@ -5,8 +5,10 @@ defmodule Snitch.Tools.OrderEmail do
   use Bamboo.EEx, path: Path.join([File.cwd!(), "lib/core/tools/mailer/email_templates"])
   alias Snitch.Tools.Mailer
   alias Snitch.Repo
+  alias Snitch.Data.Schema.Store
 
   def order_confirmation_mail(order) do
+    set_mailer_config()
```

Comment:
This doesn't look good. Why? This method would be called every time we send an email. This should happen only once.

Reply:
This was done considering multi-store. But since we have a single store, for now, it's not required here.